fixtures/notary-server.crt

```diff
@@ -1,68 +1,62 @@
 -----BEGIN CERTIFICATE-----
-MIIFWzCCA0OgAwIBAgIBAjANBgkqhkiG9w0BAQsFADBfMRowGAYDVQQDDBFOb3Rh
```

Comment:
can you try rebasing on to master?  We bumped all of the fixture certificates a little while back in https://github.com/docker/notary/pull/842, so you probably won't need this change or the next commit